app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -147,6 +147,14 @@
                 }
             }
         }
+
+        &#currently_used_for {
```

Comment:
Please avoid using id's in css selectors, try to build css selectors based on classes

Reply:
I have updated code using class.